api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -157,6 +171,48 @@ private String buildWelcomeEmailHtml(String password, User user) throws IOExcept
     return new StrSubstitutor(replaceMap).replace(string);
   }
 
+  private MandrillMessage buildIdVerificationCompleteMessage(String contactEmail, IdVerificationStatus status, String username) throws MessagingException {
+    MandrillMessage msg = new MandrillMessage();
+    RecipientAddress toAddress = new RecipientAddress();
+    toAddress.setEmail(contactEmail);
+    msg.setTo(Collections.singletonList(toAddress));
+    try {
+      String msgHtml = buildIdVerificationCompleteHtml(status, username);
+      msg.setHtml(msgHtml);
+      msg.setSubject("All of Us ID Verification Complete");
+      msg.setFromEmail(workbenchConfigProvider.get().mandrill.fromEmail);
+      return msg;
+    } catch (IOException e) {
+      throw new MessagingException("Error reading in email");
+    }
+  }
```

Comment:
optional nit: You could use the chained builder pattern for `MandrillMessage` construction, it might read better. Partial example:
```
msg.html(msgHtml)
    .subject("All of Us ID Verification Complete")
    .fromEmail(workbenchConfigProvider.get().mandrill.fromEmail)
```

Reply:
I will try that! (And refactor the other build methods as well)